inc/saiswitch.h

```diff
@@ -511,6 +511,9 @@ typedef enum _sai_switch_attr_t
     /* Custom range base value */
     SAI_SWITCH_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
+    /* Enable Broadcom switch shell [bool] (default to 0) */
+    SAI_SWITCH_ATTR_BRCM_SWITCH_SHELL_ENABLE,
```

Comment:
@lguohan  can this be made generic and not BRCM specific ? 

@cavium we support similar private extension to enable shell, making this generic would be  better.